packages/app/src/app/pages/Patron/PricingModal/PricingChoice/ChangeSubscription/index.tsx

```diff
@@ -17,20 +17,23 @@ interface Props {
   markedAsCancelled: boolean;
   cancelSubscription: () => void;
   updateSubscription: (params: { coupon: string }) => void;
-  store: any;
-  signals: any;
 }
 
 function ChangeSubscriptionComponent({
```

Comment:
Let's change this to an arrow function and just export the component directly instead of declaring then exporting with a different name at the end of the file, ie:

```typescript
export const ChangeSubscription: React.FC<IChangeSubscriptionProps> = ({
```

Reply:
Done. also verified the import reference across code. no changes are required.